### ISSUE TITLE
Is there a way to get HDMI and LCD to work?

### ISSUE THREAD

**Issue by ralfstoermer** [goodtft/LCD-show#109]
Is there a way to get HDMI and LCD-Tochscreen to work simultan?
Does the screen support fbtft?

Thanks!

**Comment by AndyGaskell** [comment]
Any ideas on this?  It would be great to have screen mirroring.

**Comment by BrayanKhosravian** [comment]
at least it should be able to deactivate the tft mode

**Comment by goodtft** [comment]
@ralfstoermer I don't understand what you said.Isn't HDMI and LCD-Touchscreen not working synchronously?

**Comment by BrayanKhosravian** [comment]
It does not work synchronously for me

**Comment by SilasMueller** [comment]
Ive got the same question. Is there no solution to this?

**Comment by webash** [comment]
I'm assuming we're unlikely to get a response here, but did anyone already in this thread get it working?

**Comment by webash** [comment]
I did; if anyone is interested, message me

**Comment by SilasMueller** [comment]
> 
> 
> I did; if anyone is interested, message me

could you post the solution here? could help others in the future too

**Comment by webash** [comment]
No worries, I wrote this to help:

https://ashley.geek.nz/2020/05/02/dual-screen-on-a-raspberry-pi-with-hdmi-goodtft-gpio-touchscreens/

**Comment by AythaE** [comment]
@webash Your blog seems to be down, it gives me a `Error establishing a database connection`. Can you share this dual screen tutorial on a different way

**Comment by webash** [comment]
@AythaE apologies for the delay, the site should be online again now :)

**Comment by elektronika-ba** [comment]
The blog is down again (page not found), but a copy on Archive.org is available:

https://web.archive.org/web/20211020140334/https://ashley.geek.nz/2020/05/02/dual-screen-on-a-raspberry-pi-with-hdmi-goodtft-gpio-touchscreens/

**Comment by webash** [comment]
Yeah sorry about that. Need to shift my blog off WordPress but y'all know how personal tech debt can be... 😅
